lectures/inflation_history.md

```diff
@@ -259,6 +273,13 @@ def process_df(df):
     df = df.loc[mask]
 
     return df
+```
+
+Now we write plotting functions so we can plot the price level, exchange rates, 
+and inflation rates, for each country of interest.
```

Comment:
I think it's good practice for us to name the funtions that will be hidden, and say roughly what they do, so readers who don't want to open the cell can still understand the code that follows.

Reply:
will add to manual.quantecon.org

Reply:
https://github.com/QuantEcon/QuantEcon.manual/commit/61566b5f675b794a893bdff28654594160d80d09